src/Sylius/Bundle/TaxonomyBundle/Form/EventListener/SetRootTaxonNameFormListener.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\TaxonomyBundle\Form\EventListener;
+
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+use Sylius\Component\Taxonomy\Model\TaxonomyInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormFactoryInterface;
+
+/**
+ * Adds the taxonomy name to the root taxon
+ *
+ * @author Andy Clyde <me@andyclyde.com>
+ */
+class SetRootTaxonNameFormListener implements EventSubscriberInterface
```

Comment:
I followed the naming schema I found in \Sylius\Bundle\TaxonomyBundle\Form\EventListener\BuildTaxonFormListener. Is there a convention in the project for how these classes should be named? Would it be best to call it \Sylius\Bundle\TaxonomyBundle\Form\EventListener\BuildTaxonomyFormListener so it matches the other class in the same folder?